core/frontend/src/render/FrameStats.ts

```diff
@@ -0,0 +1,141 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Rendering
+ */
+
+import { BeTimePoint } from "@bentley/bentleyjs-core";
+
+/** Describes timing statistics for a single rendered frame.
+ * @alpha
+ */
+export interface FrameStats {

```

Comment:
Any reason for the properties of this object to be mutable?

Reply:
Do you suggest a class with only readonly properties instead of an interface?

Reply:
No, an interface with read-only properties.